test/clj/rems/workflow/test_dynamic.clj

```diff
@@ -180,13 +185,6 @@
                  (apply-events [(assoc-in draft-saved-event [:application/field-values 41] "")])
                  (fail-command submit-command injections)))))
 
-    (testing "cannot submit when not all licenses are accepted"
```

Comment:
once this is merged, entitlements will be granted even without licenses? I guess you'll follow up with changes to the entitlement poller?

Reply:
Yes it's the remaining checkbox